src/rqt_publisher/publisher.py

```diff
@@ -170,25 +170,24 @@ def _change_publisher_topic(self, publisher_info, topic_name, new_value):
     def _change_publisher_type(self, publisher_info, topic_name, new_value):
         type_name = new_value
         # create new slot
```

Comment:
Change the 'slot' terms in the comments as well (to become 'field')?

Reply:
Thanks! I forgot the comments, I'll change them accordingly.